docs/api/events.rst

```diff
@@ -865,6 +865,23 @@ Members
     :param after: The updated member's updated info.
     :type after: :class:`Member`
 
+.. function:: on_raw_presence_update(payload)
+
+    Called when a :class:`User` updates their presence.
+    Unlike :func:`on_presence_update`, this is called regardless of the member cache.
+
+    This is called when one or more of the following things change:
+
+    - status
+    - activity
```

Comment:
These aren't needed, the non-raw event's description references them as attributes of `Member`, which doesn't apply here